_build/redirection_map

```diff
@@ -445,4 +445,5 @@
 /console/request_context /routing
 /form/action_method /forms
 /reference/requirements /setup
- /bundles/inheritance /bundles/override
+/bundles/inheritance /bundles/override
+/components/stopwatch https://symfony.com/components/Stopwatch
```

Comment:
Because this redirection map is:

```
OLD documentation page path -> NEW documentation page path
```

If we use `/components/Stopwatch` it will redirect to `https://symfony.com/doc/current/components/Stopwatch.html`.

We need to redirect to an entirely different URL and that's why we need absolute URLs.